htdocs/admin/ecm.php

```diff
@@ -49,7 +49,7 @@
 
 if (preg_match('/del_([a-z0-9_\-]+)/i', $action, $reg)) {
 	$code = $reg[1];
-	if (dolibarr_del_const($db, $code, $conf->entity) > 0) {
```

Comment:
Here $code was ECM_AUTO_TREE_ENABLED.
But i pushed recenty a pr to revert the option in ECM_AUTO_TREE_HIDDEN so now the iption is no more created during module activation (we must avoid this).
So we don't need anymore to replace this dol_delete with dol_set....

Reply:
>  i pushed recenty a pr to revert the option in ECM_AUTO_TREE_HIDDEN

You are right, you did on `develop` branch . 
This fix points to 14.0 branch. Do you want me to switch to `develop` ? There is no problem on my side.